setup.py

```diff
@@ -76,14 +75,14 @@
     url="https://github.com/ApeWorX/ethpm-types",
     include_package_data=True,
     install_requires=[
-        "pydantic>=2.4.2,<3",
-        "eth-utils>=2.1.0,<3",
+        "pydantic>=2.6.4,<3",
+        "eth-utils>=4.1.0,<5",
```

Comment:
In general, I wonder how wide our pins can be for these EF packages, I think it's best even to leave them open above (and keep the tighter pins in Ape if we need it)